openwisp_monitoring/monitoring/metrics.py

```diff
@@ -0,0 +1,42 @@
+from django.utils.translation import ugettext_lazy as _
+
+from . import settings as app_settings
+
+DEFAULT_METRICS = {
+    'ping': {
+        'label': _('Ping'),
+        'key': 'ping',
+        'field_name': 'reachable',
+        'related_fields': ['loss', 'rtt_min', 'rtt_max', 'rtt_avg'],
+    },
+    'traffic': {'label': _('Traffic'), 'key': '{key}', 'field_name': '{field}'},
+    'clients': {'label': _('Clients'), 'key': '{key}', 'field_name': 'clients'},
+}
+
+
+def deep_merge_dicts(dict1, dict2):
```

Comment:
never duplicate code, follow the DRY principle.
We already added this to openwisp-utils. Look it up and use it please.

Reply:
I didn't knew that, shall I do the same for `charts.py`?